src/Illuminate/Routing/ResponseFactory.php

```diff
@@ -87,6 +88,33 @@ public function view($view, $data = [], $status = 200, array $headers = [])
         return $this->make($this->view->make($view, $data), $status, $headers);
     }
 
+    /**
+     * Create a new compressed JSON response instance. Requires php-zip extension.
+     *
+     * @param  array|string  $data
+     * @param  int  $status
+     * @param  array  $headers
+     * @return Response
+     */
+    public function compressedJson(array|string $data = [], $status = 200, array $headers = [])
+    {
+        if (! extension_loaded('zip')) {
```

Comment:
Isn't the extension for GZip `ext-zlib` instead of `ext-zip`?

Reply:
Yeah, this needs to be `zlib`.

Reply:
Same question [here](https://github.com/laravel/framework/pull/43845/files#diff-f69e5177a6a5562428bf2fa61f2175c94b7b13a398ac6aa65bc071ebf082ba83R34).